sycl/include/sycl/handler.hpp

```diff
@@ -2314,9 +2314,12 @@ class __SYCL_EXPORT handler {
 
   template <typename KernelName = detail::auto_name, typename KernelType,
             typename PropertiesT>
-  std::enable_if_t<
-      ext::oneapi::experimental::is_property_list<PropertiesT>::value>
-  single_task(PropertiesT Props, _KERNELFUNCPARAM(KernelFunc)) {
+  __SYCL2020_DEPRECATED(
```

Comment:
`SYCL2020` deprecated means that this feature is marked as deprecated in the core SYCL 2020 specification and it will be removed in whatever next version of the core SYCL specification there will be.

However, in this case we are deprecating an overload documented by an extension - it is not related to the SYCL language version and I assume that we want it to be removed way earlier than we drop SYCL 2020 support. Therefore, regular `SYCL_DEPRECATED` macro should be used